verification/curator-service/ui/src/components/EditCase.test.tsx

```diff
@@ -36,13 +37,27 @@ it('loads and displays case to edit', async () => {
         headers: {},
     };
     mockedAxios.get.mockResolvedValueOnce(axiosCaseResponse);
+    // This is currently called twice, because the value from the case being
+    // edited is populated in the form field a split second after the page
+    // initially loads (resulting in two queries: ?url={} and ?url={fullURL}).
+    mockedAxios.get.mockResolvedValueOnce(axiosSourcesResponse);
```

Comment:
Optional: you could use `mockedAxios.get.mockResolvedValue(axiosSourcesResponse)`